coordinator/testing/src/main/java/io/stargate/it/http/MetricsTest.java

```diff
@@ -184,6 +185,22 @@ public void dropwizardMetricsPersistence() throws IOException {
     assertThat(lines).isNotEmpty();
   }
 
+  @Test
+  public void dropwizardMetricReadSizeIsPresent() throws IOException {
```

Comment:
nit: I know it's following existing naming, but since SG v2 does not use DropWizard any more (but quarkus), should just drop it and lave "metricReadSizeIsPresent()".

Reply:
The new Metric `metricReadSizehistogram` is based on Histogram class from `com.codahale.metrics. MetricRegistry` package, coming from `io.dropwizard.metrics:metrics-core:4.2.18`maven dependancy. Anyway I can update the method name, sure.